srcts/src/shiny/shinyapp.ts

```diff
@@ -827,12 +843,13 @@ class ShinyApp {
           hasOwnProperty(message, "name") &&
           hasOwnProperty(message, "status")
         ) {
-          const binding = this.$bindings[message.name];
+          const binding = this.$bindings[message.name as string];
 
-          // @ts-expect-error; TODO-barret; Could this be transformed into `.trigger(TYPE)`?
-          $(binding ? binding.el : null).trigger({
-            type: "shiny:" + message.status,
-          });
+          if (binding) {
+            $(binding.el).trigger("shiny:" + message.status);
+          } else {
+            $().trigger("shiny:" + message.status);
+          }
```

Comment:
Turned the inline `binding.el` check into two different calls. TS doesn't like `$(null)`. 

Turned the trigger call into taking a string rather than an object with `type: STRING`